website/react-magma-docs/src/pages/patterns/header.mdx

```diff
@@ -17,6 +17,7 @@ If you have a need for a header with more customization or different features in
 
 1. Your project needs to be running **React v16.8** or greater
 2. `react-magma-icons` and `react-magma-dom` must be installed as peer dependencies
+3. To install `Header` run: `npm i @cengage-patterns/header --save`
```

Comment:
Maybe we can be more explicit and explain it's a separate package. Something like:
```
Header is part of a separate package. Install it using `npm i @cengage-patterns/header --save`
```